src/docs/asciidoc/integration.adoc

```diff
@@ -3413,7 +3413,7 @@ The following table summarizes these registration behaviors:
   previous instance).
 |===
 
-The values in the preceding table are defined as enums on the `RegistrationPolicy` class. 
+The values in the preceding table are defined as enums on the `RegistrationPolicy` class.
```

Comment:
Thanks but that's unrelated to the purpose of the PR. I've fixed that in https://github.com/spring-projects/spring-framework/commit/86846507e2196ec8dbd7ea14bd9ee0f218761ddc